services/brig/test/integration/Util.hs

```diff
@@ -46,9 +46,10 @@ import qualified Data.UUID as UUID
 import qualified Data.UUID.V4 as UUID
 import qualified Test.Tasty.Cannon as WS
 
-type Brig    = Request -> Request
-type Cannon  = Request -> Request
-type Galley  = Request -> Request
+type Brig      = Request -> Request
+type Cannon    = Request -> Request
+type CargoHold = Request -> Request
```

Comment:
For consistency: Use `Cargohold` (with lowercase `h`)?

Reply:
We actually use camelcase in some cases as in `import ... CargoHold.Types.V3` since cargo hold is made up of 2 words. We do use it as `cargohold` in a few places though (binary, package-name, cargohold as a variable...)

So I think it actually makes sense to keep it camelcase here? We can then for consistency's sake indeed either move everything to camelcase or rename `CargoHold` to `Cargohold` everywhere.

Reply:
Ok, nevermind. I hadn't seen the `import ... CargoHold.Types.V3`. So it's consistent already. All good!